lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -220,6 +220,7 @@ export class UserAgentApplication {
         this.telemetryManager = this.getTelemetryManagerFromConfig(this.config.system.telemetry, this.clientId);
 
         AuthorityFactory.setKnownAuthorities(this.config.auth.validateAuthority, this.config.auth.knownAuthorities);
+        AuthorityFactory.parseAuthorityMetadata(this.config.auth.authority, this.config.auth.authorityMetadata);
```

Comment:
name? maybe setAuthorityMetadataFromJsonString

Reply:
That's a bit wordy, I think parseAuthorityMetadata is fine.